src/NRedisStack/Json/IJsonCommands.cs

```diff
@@ -205,6 +205,17 @@ public interface IJsonCommands
     /// <remarks><seealso href="https://redis.io/commands/json.set"/></remarks>
     bool Set(RedisKey key, RedisValue path, RedisValue json, When when = When.Always);
 
+    /// <summary>
+    /// Set json file from the provided file Path.
+    /// </summary>
+    /// <param name="key">The key.</param>
+    /// <param name="path">The path to set within the key.</param>
+    /// <param name="filePath">The path of the file to set.</param>
+    /// <param name="when">When to set the value.</param>
+    /// <returns>The disposition of the command</returns>
+    /// <remarks><seealso href="https://redis.io/commands/json.set"/></remarks>
+    bool SetFile(RedisKey key, RedisValue path, string filePath, When when = When.Always);
```

Comment:
Is there any way to have a default path value of "." (similar to redis-py)?

Reply:
the path in JSON.SET Command is not optional so I thought it shouldn't be optional here either.
What about "$" as default value?

Reply:
In python default value is ".". I notice that in node - there is no default. The [documentation](https://redis.io/commands/json.set/) says that the default should be "$", so we should do that.